### PR TITLE
add process.hrtime.bigint() to node v10

### PR DESCRIPTION
**Pull request by jstewmon**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://nodejs.org/api/process.html#process_process_hrtime_bigint

### CODE REVIEW

**Comment by jstewmon** [comment]
Amending the commit seems to have awoken travis.

File: types/node/v10/package.json [hunk 1]
```diff
@@ -6,6 +6,11 @@
             "*": [
                 "ts3.1/*"
             ]
+        },
+        ">=3.2.0-0": {
```
Comment (SimonSchick) [inline types/node/v10/package.json]: I don't think splitting typings into even more versions is a good idea, bumping up 3.1 to 3.2 should be fine.

Comment (jstewmon) [reply types/node/v10/package.json]: @SimonSchick thanks for the feedback. I've added a new commit to consolidate the versions.

**Comment by jstewmon** [comment]
The build is failing re: https://github.com/DefinitelyTyped/DefinitelyTyped/commit/b926920fcdf821bc6d3daa890834bb47b8e46a00#commitcomment-32383405

**Comment by Flarna** [comment]
@jstewmon Why do you add this only for node 10? BitInt is also supported for node 11. I think latest definitions should include the latest and greatest definitions.

**Comment by jstewmon** [comment]
@Flarna it was just an oversight on my part. I've updated the PR to cover node 11. Thanks for the review!

**Review by SimonSchick — APPROVED** [review]

**Review by Flarna — APPROVED** [review]

**Comment by jstewmon** [comment]
The build errors do not appear to be related to the changes in this PR. There's a mix of lint errors and compiler errors in various packages, which appear to be in need of updating for compatibility with the current compiler and lint rules.

**Comment by Flarna** [comment]
@jstewmon see https://github.com/Microsoft/TypeScript/issues/29898

**Review by BrunoScheufler — APPROVED** [review]

**Pull request merged**